src/uu/link/src/link.rs

```diff
@@ -8,14 +8,42 @@
 #[macro_use]
 extern crate uucore;
 
+use clap::{App, Arg, ArgMatches};
 use std::fs::hard_link;
 use std::io::Error;
-use std::path::Path;
-use uucore::InvalidEncodingHandling;
+use std::path::{Path, PathBuf};
 
-static SYNTAX: &str = "[OPTIONS] FILE1 FILE2";
-static SUMMARY: &str = "Create a link named FILE2 to FILE1";
-static LONG_HELP: &str = "";
+static VERSION: &str = env!("CARGO_PKG_VERSION");
+static ABOUT: &str = "Call the link function to create a link named FILE2 to an existing FILE1.";
+
+pub mod options {
+    pub static FILE1: &str = "FILE1";
+    pub static FILE2: &str = "FILE2";
+}
+
+#[derive(Debug)]
+struct Options {
+    old: PathBuf,
+    new: PathBuf,
+}
+
+impl Options {
```

Comment:
are you sure this is necessary?

Reply:
No, I will remove this and use the technique @jhscheer suggested below.